src/parser/parsers/abilities.py

```diff
@@ -26,6 +26,7 @@ def run(self):
             ability_data = {
                 'Key': ability_key,
                 'Name': self.localizations.get(ability_key, None),
+                'IsDisabled': ability.get('m_bIsDisabled', True),
```

Comment:
similarly, why is this default to `True`?

Reply:
Just noticed every ability is disabled because this should be `m_bDisabled`. I still don't think this should default to True though

Reply:
Oh, that explains my failed state of reference then

Reply:
Decompiled abilities with Disabled=:
* true, 59 - looks correct, all of these are not in game
* false, 19 - several in here are definitely enabled, but some i'm pretty certain are not in game
In parsed data when I set default=none, ~180 are null, including ones I definitely know are in game like infernus' afterburn

Items:
* true, 72 - never heard of any of these so looks correctly disabled
* false, 17 - they all look correctly enabled, but there should be more than 17
* null, 117 - some of these are in game like refresher and some are not

Idek anymore, can't find any pattern. I pushed the commit to default both to None if you want to take a look.

Reply:
I'd be interested in any examples you found where the flag wasn't found, but you say are not in game

Reply:
> I'd be interested in any examples you found where the flag wasn't found, but you say are not in game

All of bomber's abilities, i.e. `ability_bomber_ability03`, bomber hero's flags are:
```
        "InDevelopment": true,
        "InHeroLabs": false,
        "IsDisabled": true,
        "IsRecommended": false,
        "IsSelectable": true,
```
So I suppose the definition of what's "in game" is a bit blurry, at least for hero abilities.
Also notably `ability_death_tax`, which isn't attached to any hero in decompiled data so I presume its an item.

Reply:
I guess it depends on how they use their disabled flags. From your evidence, the majority of items/abilities that don't have a disabled flag seem to be in the game, so i think defaulting to False is correct here

Reply:
Well, it seems that the problem is now that regardless of how we default it, there will be items that are enabled that aren't attainable in game. Is there another local file perhaps that determines if they are attainable, since clearly its not using the IsDisabled key? Your right though that defaulting to False is definitely the lesser evil- I'll set the default back to False and we can merge it, then hopefully find something more concrete for longevity sake.